### ISSUE TITLE
Trouble with copying a code by Code Button

### ISSUE THREAD

**Issue by Programtica** [KevinBatdorf/code-block-pro#300]
Hello @KevinBatdorf ,

When I tried to copy code made in C++/Python/Java to an external compiler using the Copy button, the formatting is broken, it looks like the lines are separated by a tabulator…. Is it possible to copy it properly?

I am using a WordPress 6.4.3, with Gutenberg and Kadence.

https://programtica.edu.pl/rok-przestepny/ - my post with code
https://imgur.com/a/ORrIW7q - code in my page
https://imgur.com/H9EMX1q - code in GDB Online
https://www.onlinegdb.com/ - compiler (GDB Online)

**Comment by KevinBatdorf** [comment]
Hey, can you look at this issue which is the same situation?

https://github.com/KevinBatdorf/code-block-pro/issues/249

Do you have this plugin "[Flying Scripts](https://wordpress.org/plugins/flying-scripts/)"? If not, can you deactivate other plugins and see which one is comflicting?

**Comment by Programtica** [comment]
No, I don't have this plugin... So I must deactivate 1 by 1 plugin and check, is everything okey? And if not, disable it?

**Comment by KevinBatdorf** [comment]
Yeah. But easier to disable half, reload, then half, then narrow it down to one. And if you find the one causing the issue, I can install it and see if I can add something to override it or fix it, or I can try to contact the dev too. 

It could also be your theme as well, but we just need to narrow it down first.

**Comment by Programtica** [comment]
I will see this in next days 👍

**Comment by Programtica** [comment]
@KevinBatdorf I discovered a trouble... it was caused an Orphans (Sierotki) plugin...

https://wordpress.org/plugins/sierotki/

**Comment by KevinBatdorf** [comment]
I opened an issue there and lets wait to hear back https://github.com/iworks/sierotki/issues/6
